projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx

```diff
@@ -103,9 +126,15 @@ export class DotDateRangeComponent {
         }
     }
 
+    @Watch('presets')
+    presetsWatch(): void {
+        this.presets = Array.isArray(this.presets) ? this.presets : this.defaultPresets;
```

Comment:
What happen if we pass an array but it doesn't have the proper format?

Reply:
leaves the preset dropdown empty